recipes/poco/all/patches/1.11.0.patch

```diff
@@ -29,14 +33,59 @@
  	target_compile_definitions(Foundation PUBLIC POCO_UNBUNDLED)
  endif(POCO_UNBUNDLED)
  
---- NetSSL_Win/CMakeLists.txt
-+++ NetSSL_Win/CMakeLists.txt
-@@ -21,7 +21,7 @@
+diff --git a/NetSSL_Win/CMakeLists.txt b/NetSSL_Win/CMakeLists.txt
+index 3e2b5ff..2ab7590 100644
+--- a/NetSSL_Win/CMakeLists.txt
++++ b/NetSSL_Win/CMakeLists.txt
+@@ -21,7 +21,7 @@ set_target_properties(NetSSLWin
  	DEFINE_SYMBOL NetSSL_Win_EXPORTS
  )
  
 -target_link_libraries(NetSSLWin PUBLIC Poco::Net Poco::Util Crypt32.lib)
-+target_link_libraries(NetSSLWin PUBLIC Poco::Net Poco::Util Crypt32.lib ws2_32 )
++target_link_libraries(NetSSLWin PUBLIC Poco::Net Poco::Util crypt32 ws2_32)
  target_include_directories(NetSSLWin
  	PUBLIC
  		$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+diff --git a/PageCompiler/CMakeLists.txt b/PageCompiler/CMakeLists.txt
+index d80fe73..48379eb 100644
+--- a/PageCompiler/CMakeLists.txt
++++ b/PageCompiler/CMakeLists.txt
+@@ -23,5 +23,6 @@ install(
+ 	LIBRARY DESTINATION lib${LIB_SUFFIX}
+ 	ARCHIVE DESTINATION lib${LIB_SUFFIX}
+ 	RUNTIME DESTINATION bin
++	BUNDLE DESTINATION bin
```

Comment:
Has this problem/patch been brought to the attention of upstream?

Reply:
???

Reply:
I rephrase, has the poco project been notified of this problem? To avoid having to carry this patch for next poco releases.

Reply:
Yes, i made a pull-request for the lib owner. He will add in future releases (i think 1.11.1). In the new version this patch will be unecessary.